components/sharedfiles.js

```diff
@@ -71,14 +82,25 @@ SteamCommunity.prototype.postSharedFileComment = function(userID, sharedFileId,
 		"form": {
 			"comment": message,
 			"count": 10,
+			"json": 1,
 			"sessionid": this.getSessionID()
-		}
+		},
+		"json": true
 	}, function(err, response, body) {
 		if (!callback) {
 			return;
 		}
 
-		callback(err);
+		if (err) {
+			callback(err);
+			return;
+		}
+
+		if (body.success) {
+			callback(null);
+		} else {
+			callback(new Error(body.error));
+		}
```

Comment:
Same question as above.

Reply:
Also tested; Steam returns the following: `{ success: false, error: 'There was a problem posting your comment. Please try again.' }`